app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -14,4 +14,25 @@ def provider_collection(manager)
   def provider_object(connection = nil)
     (connection || connection_source.connect).api.projects.find(manager_ref)
   end
+
+  REFRESH_ON_TOWER_SLEEP = 1.second
+  def refresh_in_provider
+    with_provider_object do |project|
+      return unless project.can_update?
+
+      project_update = project.update
+
+      # this is really just a quick hack. We should do this properly once
+      # https://github.com/ManageIQ/manageiq/pull/14405 is merged
+      log_header = "updating project #{project.id} (#{self.class.name} #{id})"
+      _log.info "#{log_header}..."
+      Timeout.timeout(5.minutes) do
```

Comment:
I would sleep better if we would make this configurable, this should be based on the size of project inside of the Ansible Tower, right?

But can be made configurable in a followup PR I suppose